libafl_bolts/src/rands/mod.rs

```diff
@@ -96,6 +96,13 @@ pub fn fast_bound(rand: u64, n: NonZeroUsize) -> usize {
     (mul >> 64) as usize
 }
 
+#[inline]
+#[must_use]
+fn fast_bound_usize(rand: u64, n: usize) -> usize {
```

Comment:
Anything modulo 0 doesn't really make sense?
But if we want to introduce this, and say the result is 0 we can also just remove the other function and document it

Reply:
I didn't say it's module 😛 
It is a function just for my use case that just works exactly the same as `fast_bound` and defined for 0 at the same time.

for cases like this: https://github.com/AFLplusplus/LibAFL/pull/2589#discussion_r1800264447